onedal/.clang-format

```diff
@@ -0,0 +1,155 @@
+---

```

Comment:
Is this config similar to what we have for oneAPI interfaces of DAL?

Reply:
Yes, copy-paste